### ISSUE TITLE
ValueError: facet_config["plot"]["scale"] must a string

### ISSUE THREAD

**Issue by jagjeet-singh** [lvwerra/jupyterplot#5]
Thank you for such a useful tool.

Just recently, I started facing the error `ValueError: facet_config["plot"]["scale"] must a string`

Just using the snippet from the README
```
from jupyterplot import ProgressPlot
import numpy as np

pp = ProgressPlot()
for i in range(1000):
    pp.update(np.sin(i / 100))
pp.finalize()
```
I get
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-7-c92dc3164c40> in <module>
      2 import numpy as np
      3 
----> 4 pp = ProgressPlot()
      5 for i in range(1000):
      6     pp.update(np.sin(i / 100))

/usr/local/lib/python3.6/dist-packages/jupyterplot/jupyterplot.py in __init__(self, plot_names, line_names, line_colors, x_lim, y_lim, x_label, x_iterator, height, width, display_fn, debug)
    120             self.update = self._update_with_x
    121 
--> 122         self._setup_plot()
    123 
    124     def _update_with_iter(self, y):

/usr/local/lib/python3.6/dist-packages/jupyterplot/jupyterplot.py in _setup_plot(self)
    216             xaxis_config=xaxis_config,
    217             display_fn=self.display_fn,
--> 218             debug=self.debug,
    219         )

/usr/local/lib/python3.6/dist-packages/lrcurve/plot_learning_curve.py in __init__(self, display_fn, debug, **kwargs)
    143             display_id=True
    144         )
--> 145         self.reconfigure(**kwargs)
    146 
    147     def reconfigure(self,

/usr/local/lib/python3.6/dist-packages/lrcurve/plot_learning_curve.py in reconfigure(self, height, width, mappings, line_config, facet_config, xaxis_config)
    166         """
    167         height = len(facet_config) * 200 + 90 if height is None else height
--> 168         validate_settings(height, width, mappings, line_config, facet_config, xaxis_config)
    169 
    170         self._settings.update({

/usr/local/lib/python3.6/dist-packages/lrcurve/plot_learning_curve.py in validate_settings(height, width, mappings, line_config, facet_config, xaxis_config)
     39             raise ValueError(f'facet_config["{facet_key}"]["limit"] must be a list with length two')
     40         if 'scale' not in facet_def or not isinstance(facet_def['scale'], str):
---> 41             raise ValueError(f'facet_config["{facet_key}"]["scale"] must a string')
     42         if facet_def['scale'] not in {'log10', 'linear'}:
     43             raise ValueError(f'the facet scale {facet_def["scale"]} must either log10 or linear')

ValueError: facet_config["plot"]["scale"] must a string
```

**Comment by skyiwalker** [comment]
same problem here

**Comment by lvwerra** [comment]
Hi, thanks for raising the issue. I know that lrcurve (the underlying plotting library) had a major update which could have caused the issue. Could you report which version of lrcurve is installed on your environment? The command `pip freeze` should show you that.

**Comment by skyiwalker** [comment]
In my case,
lrcurve==2.1.0. Thank you.

**Comment by iotic-labs-markwharton** [comment]
I'm getting the same error, too.
I have 
`lrcurve==2.1.0`

**Comment by anag004** [comment]
Installing an older version of `lrcurve` worked for me. After installing `jupyterplot` using pip

`
pip uninstall lrcurve
pip install lrcurve==1.1.0
`

This fixed the problem.

**Comment by lvwerra** [comment]
This should be fixed with commit 07500008e063d8703011abc5b91a4e85167a9ac2. The version has been bumped up to jupyterplot 0.0.3 so if you upgrade with `pip install --upgrade jupyterplot` the issue should go away.
